assets/js/components/wp-dashboard/wp-dashboard-impressions.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * WPDashboardImpressions component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { useEffect } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { STORE_NAME } from '../../modules/search-console/datastore/constants';
```

Comment:
I knew I had seen it somewhere 🙂  let's update this as mentioned previously as it is our convention to do so any time we're not in the context of a specific module's store. I agree with @tofumatt that we should export these as constants rather than relying on this convention, but let's update it anyways for consistency. 

```suggestion
import { STORE_NAME as MODULES_SEARCH_CONSOLE } from '../../modules/search-console/datastore/constants';
```